app/scripts/controllers/network/createPocketClient.js

```diff
@@ -0,0 +1,96 @@
+const mergeMiddleware = require('json-rpc-engine/src/mergeMiddleware')
+const createScaffoldMiddleware = require('json-rpc-engine/src/createScaffoldMiddleware')
+const createBlockReRefMiddleware = require('eth-json-rpc-middleware/block-ref')
+const createRetryOnEmptyMiddleware = require('eth-json-rpc-middleware/retryOnEmpty')
+const createBlockCacheMiddleware = require('eth-json-rpc-middleware/block-cache')
+const createInflightMiddleware = require('eth-json-rpc-middleware/inflight-cache')
+const createBlockTrackerInspectorMiddleware = require('eth-json-rpc-middleware/block-tracker-inspector')
+const providerFromMiddleware = require('eth-json-rpc-middleware/providerFromMiddleware')
+const createPocketMiddleware = require('json-rpc-pocket')
+const BlockTracker = require('eth-block-tracker')
+const devid = 'DEVVGQ8VfHgBBet8CyowHcN'
+
+module.exports = createPocketClient
+
+function createPocketClient ({ network }) {
+  const networkIDs = getNetworkIds({ network })
+  const pocketMiddleware = createPocketMiddleware(devid, {
+    netID: networkIDs.netId,
+    network: networkIDs.ticker,
+  })
+  const pocketProvider = providerFromMiddleware(pocketMiddleware)
+  const blockTracker = new BlockTracker({ provider: pocketProvider })
+
+  const networkMiddleware = mergeMiddleware([
+    createNetworkAndChainIdMiddleware({ network }),
+    createBlockCacheMiddleware({ blockTracker }),
+    createInflightMiddleware(),
+    createBlockReRefMiddleware({ blockTracker, provider: pocketProvider }),
+    createRetryOnEmptyMiddleware({ blockTracker, provider: pocketProvider }),
+    createBlockTrackerInspectorMiddleware({ blockTracker }),
+    pocketMiddleware,
+  ])
+  return { networkMiddleware, blockTracker }
+}
+
+function getNetworkIds ({ network }) {
+  let chainId
+  let netId
+  let ticker
+  switch (network) {
+    case 'mainnet':
+      netId = '1'
+      chainId = '0x01'
+      ticker = "ETH"
+      break
+    case 'ropsten':
+      netId = '3'

```

Comment:
There are constants for each network ID https://github.com/poanetwork/nifty-wallet/blob/master/app/scripts/controllers/network/enums.js#L14-L24. Please use them instead of plain strings